tutorials/tensordictmodule.ipynb

```diff
@@ -0,0 +1,1075 @@
+      "    device=cpu,\n",
+      "    is_shared=False)\n",
+      "TensorDict(\n",
+      "    fields={\n",
+      "        action: Tensor(torch.Size([3, 4]), dtype=torch.float32),\n",
+      "        hidden: Tensor(torch.Size([3, 4]), dtype=torch.float32),\n",
+      "        loc: Tensor(torch.Size([3, 4]), dtype=torch.float32),\n",
+      "        observation: Tensor(torch.Size([3, 4]), dtype=torch.float32),\n",
+      "        sample_log_prob: Tensor(torch.Size([3, 1]), dtype=torch.float32),\n",
+      "        scale: Tensor(torch.Size([3, 4]), dtype=torch.float32)},\n",
+      "    batch_size=torch.Size([3]),\n",
+      "    device=cpu,\n",
+      "    is_shared=False)\n",
+      "TensorDict(\n",
+      "    fields={\n",
+      "        action: Tensor(torch.Size([3, 4]), dtype=torch.float32),\n",
+      "        hidden: Tensor(torch.Size([3, 4]), dtype=torch.float32),\n",
+      "        loc: Tensor(torch.Size([3, 4]), dtype=torch.float32),\n",
+      "        observation: Tensor(torch.Size([3, 4]), dtype=torch.float32),\n",
+      "        sample_log_prob: Tensor(torch.Size([3, 1]), dtype=torch.float32),\n",
+      "        scale: Tensor(torch.Size([3, 4]), dtype=torch.float32),\n",
+      "        state_action_value: Tensor(torch.Size([3, 1]), dtype=torch.float32)},\n",
+      "    batch_size=torch.Size([3]),\n",
+      "    device=cpu,\n",
+      "    is_shared=False)\n"
+     ]
+    }
+   ],
+   "source": [
+    "from torchrl.modules import (\n",
+    "    MLP,\n",
+    "    ActorCriticOperator,\n",
+    "    NormalParamWrapper,\n",
+    "    TanhNormal,\n",
+    "    ValueOperator,\n",
+    ")\n",
+    "from torchrl.modules.tensordict_module import ProbabilisticActor\n",
+    "\n",
+    "module_hidden = torch.nn.Linear(4, 4)\n",
+    "td_module_hidden = TensorDictModule(\n",
+    "    module=module_hidden,\n",
+    "    in_keys=[\"observation\"],\n",
+    "    out_keys=[\"hidden\"],\n",
+    ")\n",
+    "module_action = NormalParamWrapper(torch.nn.Linear(4, 8))\n",
+    "module_action = TensorDictModule(\n",
+    "    module_action, in_keys=[\"hidden\"], out_keys=[\"loc\", \"scale\"]\n",
+    ")\n",
+    "td_module_action = ProbabilisticActor(\n",
+    "    module=module_action,\n",
+    "    dist_param_keys=[\"loc\", \"scale\"],\n",
+    "    out_key_sample=[\"action\"],\n",
+    "    distribution_class=TanhNormal,\n",
+    "    return_log_prob=True,\n",
+    ")\n",
+    "module_value = MLP(in_features=8, out_features=1, num_cells=[])\n",
+    "td_module_value = ValueOperator(\n",
+    "    module=module_value,\n",
+    "    in_keys=[\"hidden\", \"action\"],\n",
+    "    out_keys=[\"state_action_value\"],\n",
+    ")\n",
+    "td_module = ActorCriticOperator(\n",
+    "    td_module_hidden, td_module_action, td_module_value\n",
+    ")\n",
+    "td = TensorDict(\n",
+    "    {\"observation\": torch.randn(3, 4)},\n",
+    "    [\n",
+    "        3,\n",
+    "    ],\n",
+    ")\n",
+    "print(td)\n",
```

Comment:
Add a comment in the prints, otherwise we don't understand what is being printed